libraries/AC_Avoidance/AP_OADatabase.h

```diff
@@ -31,15 +31,26 @@ class AP_OADatabase {
         Vector3f pos;           // position of the object as an offset in meters from the EKF origin
         uint32_t timestamp_ms;  // system time that object was last updated
         float radius;           // objects radius in meters
+
+        // Unique ID, assigned by the source, not used by all sources
+        uint32_t id;
```

Comment:
I guess this increases memory usage by about 1k bytes (5 bytes for each of 100 items).  I'm OK with this but FYI